City-of-Helsinki/drupal-helfi-platform-config
# UHF-2526 Translations

How to test: 
- Run `composer require drupal/helfi_platform_config:dev-UHF-2526_translations`
- Run `make new`
- Run `make drush-cex`
- Check if there is LESS overridden configuration files. 🤞 

There's also these:
https://github.com/City-of-Helsinki/drupal-helfi/pull/193
https://github.com/City-of-Helsinki/drupal-helfi-sote/pull/81
https://github.com/City-of-Helsinki/drupal-helfi-kymp/pull/80
https://github.com/City-of-Helsinki/drupal-helfi-strategia/pull/36
